src/main/java/decodex/ui/messages/ErrorMessages.java

```diff
@@ -34,4 +34,6 @@
     public static final String RECIPE_FILE_DELETE_FAILED_MESSAGE = "Failed to delete the corresponding recipe file";
     public static final String FAILED_TO_LOAD_RECIPE_FILE_MESSAGE = " failed to be loaded into Decodex";
     public static final String INVALID_RECIPE_COMMAND = "Invalid recipe command";
+    public static final String ADDITIONAL_PARAMETER_AFTER_RECIPE_NAME_MESSAGE =
+            "Additional argument found after recipe name";

```

Comment:
A general exception like `too many arguments` would be better? I also realized we have a few `too many ...` exception messages

Reply:
That is literally `too many of too many`, but yes, will make changes to a more general exception.